src/app/CommandResponseSender.cpp

```diff
@@ -206,6 +206,10 @@ void CommandResponseSender::OnInvokeCommandRequest(Messaging::ExchangeContext *
     VerifyOrDieWithMsg(ec != nullptr, DataManagement, "Incoming exchange context should not be null");
     VerifyOrDieWithMsg(mState == State::ReadyForInvokeResponses, DataManagement, "state should be ReadyForInvokeResponses");
 
+    if (ec->GetSessionHandle()->AllowsLargePayload())
+    {
+        mUseLargePayloadBuffer = true;
```

Comment:
This member seems to be write-only.  Why is it here?

Reply:
This was residual from the previous logic of using an out-of-band mechanism. Should not have been here.